packages/backend/src/accounting/transfers.ts

```diff
@@ -52,33 +57,48 @@ export async function createTransfers(
       id: toTigerbeetleId(transfers[i].id || uuid()),
       debit_account_id: toTigerbeetleId(transfer.sourceAccountId),
       credit_account_id: toTigerbeetleId(transfer.destinationAccountId),
-      amount: transfer.amount,
-      user_data: BigInt(0),
-      reserved: TRANSFER_RESERVED,
-      code: 0,
+      user_data: 0n,
+      reserved: 0n,
+      pending_id: transfer.pendingId ? toTigerbeetleId(transfer.pendingId) : 0n,
+      timeout: transfer.timeout ? transfer.timeout * BigInt(10e6) : 0n, // ms -> ns
+      ledger: transfer.ledger,
+      code: ACCOUNT_TYPE,
       flags,
-      timeout: transfer.timeout ? transfer.timeout * BigInt(10e6) : BigInt(0), // ms -> ns
-      timestamp: BigInt(0)
+      amount: transfer.amount,
+      timestamp: 0n
     })
   }
   const res = await deps.tigerbeetle.createTransfers(tbTransfers)
   for (const { index, code } of res) {
     switch (code) {
       case CreateTransferErrorCode.linked_event_failed:
         break
+      // 1st phase
+      // TODO @jason: This needs to be removed: =========>
+      case 13: //debit_account_not_found,
+        return { index, error: TransferError.UnknownSourceAccount }
+      case 14: //credit_account_not_found,
+        return { index, error: TransferError.UnknownDestinationAccount }
+      case 17: //exists_with_different_flags,
+        return { index, error: TransferError.TransferExists }
+      case 19: //exists_with_different_credit_account_id,
+        return { index, error: TransferError.TransferExists }
+      case 32: //exceeds_credits
+        return { index, error: TransferError.InsufficientBalance }
+      case 33: //exceeds_debits
+        return { index, error: TransferError.InsufficientDebitBalance }
+      // TODO @jason: stop ==============================>
```

Comment:
Is this meant to be here?

Reply:
Yes, once we have all the issues ironed out with LSM, I will use that release and remove this code.
@kprotty